### PR TITLE
feat: add function for getting a list of attributes for customers

### PR DESCRIPTION
**Pull request by omBratteng**
Adds a function for getting a list of customers attributes
https://customer.io/docs/api/app/#operation/getPeopleById

### CODE REVIEW

**Review by rebelchris — APPROVED** [review]

**Review by nensidosari — APPROVED** [review]

File: lib/api.ts [hunk 1]
```diff
@@ -158,6 +158,22 @@ export class APIClient {
 
     return this.request.get(`${this.apiRoot}/customers/${id}/attributes?id_type=${idType}`);
   }
+
+  getAttributesBatch(ids: string[]) {
+    if (isEmpty(ids)) {
```
Comment (mike-engel) [inline lib/api.ts:163]: `isEmpty` needs to be updated to handle arrays

File: test/api.ts [hunk 2]
```diff
@@ -499,3 +499,33 @@ test('#getAttributes: success with type email', (t) => {
     ),
   );
 });
+
+test('#getAttributesBatch: fails without customerId', (t) => {
```
Comment (mike-engel) [inline test/api.ts:503]:
```suggestion
test('#getAttributesBatch: fails without customer ids', (t) => {
```

File: test/api.ts [hunk 3]
```diff
@@ -499,3 +499,33 @@ test('#getAttributes: success with type email', (t) => {
...
+  t.throws(() => (t.context.client.getAttributesBatch as any)(Array(101).fill(1)), {
+    message: 'Maximum of 100 customer IDs allowed',
+  });
+  t.falsy((t.context.client.request.post as SinonStub).calledWith(`${RegionUS.apiUrl}/customers/attributes`));
+});
+
+test('#getAttributesBatch: success with default type id', (t) => {
+  sinon.stub(t.context.client.request, 'post');
+  t.context.client.getAttributesBatch(['1']);
+  t.truthy((t.context.client.request.post as SinonStub).calledWith(`${RegionUS.apiUrl}/customers/attributes`));
```
Comment (mike-engel) [inline test/api.ts:530]:
```suggestion
  t.true((t.context.client.request.post as SinonStub).calledWith(`${RegionUS.apiUrl}/customers/attributes`));
```

File: test/api.ts [hunk 4]
```diff
@@ -499,3 +499,33 @@ test('#getAttributes: success with type email', (t) => {
...
+
+test('#getAttributesBatch: fails if length of ids is greater than 100', (t) => {
+  sinon.stub(t.context.client.request, 'post');
+  t.throws(() => (t.context.client.getAttributesBatch as any)(Array(101).fill(1)), {
+    message: 'Maximum of 100 customer IDs allowed',
+  });
+  t.falsy((t.context.client.request.post as SinonStub).calledWith(`${RegionUS.apiUrl}/customers/attributes`));
+});
+
+test('#getAttributesBatch: success with default type id', (t) => {
```
Comment (mike-engel) [inline test/api.ts:527]: Another assertion is needed to make sure the ids were passed through correctly

**Review by mike-engel — CHANGES_REQUESTED** [review]
Thanks for this PR, and sorry for the delay in review.

Another test should be added that tests against an empty array

**Comment by omBratteng** [comment]
@mike-engel I no longer work at dailydev, so I do not have access to the fork and am unable to do the changes.
